containers/base/Dockerfile

```diff
@@ -87,6 +87,7 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
         pandas==0.22.0 \
         pandocfilters==1.4.2 \
         pillow==5.0.0 \
+        pip==9.0.3 \
```

Comment:
Just curious, newer pip versions cause issues or is this just speculative?

Reply:
The newer pip version broke the build.

There is a warning during our build about uninstalling a distutils package (it upgrades "futures" from 3.0.3 to 3.0.5) being only a partial uninstall.

With pip 10, that warning was converted to an error.